deploy/kubernetes/helm/che/templates/configmap.yaml

```diff
@@ -85,13 +85,15 @@ data:
 {{- if .Values.workspaceDefaultRamLimit }}
   CHE_WORKSPACE_DEFAULT_MEMORY_LIMIT_MB: {{ .Values.workspaceDefaultRamLimit }}
 {{- end }}
-{{- if and .Values.che .Values.che.workspace }}
-  {{- if .Values.che.workspace.devfileRegistryUrl }}
-  CHE_WORKSPACE_DEVFILE__REGISTRY__URL: {{ .Values.che.workspace.devfileRegistryUrl | quote}}
-  {{- end }}
-  {{- if .Values.che.workspace.pluginRegistryUrl }}
-  CHE_WORKSPACE_PLUGIN__REGISTRY__URL: {{ .Values.che.workspace.pluginRegistryUrl | quote}}
-  {{- end }}
+{{- if .Values.cheDevfileRegistry.url }}
+  CHE_WORKSPACE_DEVFILE__REGISTRY__URL: {{ .Values.cheDevfileRegistry.url | quote }}
+{{- else if .Values.cheDevfileRegistry.deploy }}
+  CHE_WORKSPACE_DEVFILE__REGISTRY__URL: http://{{ printf .Values.global.cheDevfileRegistryUrlFormat .Release.Namespace .Values.global.ingressDomain }}
```

Comment:
how it will work if che would be configured to work over https?

Reply:
hmm, good question. It probably will not work. I guess we would need to have registries also on https then. I'll look at that.

Reply:
fixed https://github.com/eclipse/che/pull/13890/commits/af3eaeb980611b131e7559c720797e67da519340
I'd like to avoid duplication of `print`, but I don't know how to do it to keep readability.